app/assets/stylesheets/administrate/components/_field-unit.scss

```diff
@@ -15,3 +15,23 @@
 .field-unit__field {
   @include span-columns(5 of 12);
 }
+
+.field-unit--has-one {
```

Comment:
To keep the CSS reusable, I wonder if we don't tie this to `has-one`, but rather name this simply as a nested unit?